src/views/meat-baits-in-traps.njk

```diff
@@ -1,21 +1,57 @@
 {% extends "_base.njk" %}
 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">
-    Did you use meat bait in your traps?
-  </h1>
+
+  {% if model.meatBaitsError %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [{
+        text: "Select yes if you used meat baits.",
+        href: "#meat-baits-error"
+      }]
+    }) }}
+  {% endif %}
+
   <div class="govuk-grid-row">
 
+    <form method="post">
+
+      {{ govukRadios({
+        classes: "govuk-radios--inline",
+        idPrefix: "meat-baits-used",
+        name: "meatBaitsUsed",
+        fieldset: {
+          legend: {
+            text: "Did you use meat baits in your traps?",
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        errorMessage: {
+        text: "Select yes if you used meat baits."
+        } if model.meatBaitsError,
+        items: [
+          {
+            value: "yes",
+            text: "Yes"
+          },
+          {
+            value: "no",
+            text: "No"
+          }
+        ]
```

Comment:
You'll need a `checked` field (or whatever it's called) so that if a user comes back to this page their previous selection is remembered.